src/plugins/discover/public/hooks/use_es_doc_search.ts

```diff
@@ -109,3 +69,44 @@ export function useEsDocSearch({
 
   return [status, hit, requestData];
 }
+
+/**
+ * helper function to build a query body for Elasticsearch
+ * https://www.elastic.co/guide/en/elasticsearch/reference/current//query-dsl-ids-query.html
+ */
+export function buildSearchBody(
```

Comment:
just a little style change, moving the helper function to the bottom, so when scrolling the doc, the important function is displayed first